ods_ci/tests/Tests/400__ods_dashboard/416__ods_dashboard_settings_accelerator_profiles.robot

```diff
@@ -0,0 +1,167 @@
+*** Settings ***
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsAcceleratorProfiles.resource
+Resource        ../../Resources/CLI/AcceleratorProfiles/AcceleratorProfiles.resource
+Resource        ../../Resources/ODS.robot
+Suite Setup     Setup Settings Accelerator Profiles
+Suite Teardown  Teardown Settings Accelerator Profiles
+
+
+*** Variables ***
+${ACC_DISPLAY_NAME}=    qe_accelerator_profile_
+${ACC2_DISPLAY_NAME}=    Test: Modify Accelerator Profile 2
+${ACC3_DISPLAY_NAME}=    Test: Delete Tolerations from AP 3
+${ACC4_DISPLAY_NAME}=    Test: Disable AP 4 from Edit Detail View
+${ACC5_DISPLAY_NAME}=    Test: Enable AP 5 from Edit Detail View
+${ACC6_DISPLAY_NAME}=    Test: Disable AP 6 from Administration View
+${ACC7_DISPLAY_NAME}=    Test: Enable AP 7 from Administration View
+${ACC8_DISPLAY_NAME}=    Test: Delete AP 8 from grid
+${ACC_NAME}=    qeacceleratorprofile
+${ACC_IDENTIFIER}=    nvidia.com/gpu
+${ACC_DESCRIPTION}=    Create Accelerator Profile
+${ACC_DESCRIPTION2}=    Modify Accelerator Profile
+${ACC_ENABLED}=    True
+${ACC_TOLERATION_OPERATOR}=    Exists
+${ACC_TOLERATION_EFFECT}=    PreferNoSchedule
+${ACC_TOLERATION_KEY}=    my_key
+${ACC_TOLERATION_VALUE}=    my_value
+${ACC_TOLERATION_SECONDS}=    15
+
+
+*** Test Cases ***
+Create An Accelerator Profile From Accelerator Profiles Administration UI
+    [Documentation]    Create an Accelerator Profile instance from the Administration UI and verify it's content
+    [Tags]  RHOAIENG-3349
+    ...     Smoke
```

Comment:
Normally we add the tests to Smoke when they have been tested for a while and they are proven to be stable. Do you mind adding it to Sanity and Tier1 now?

Reply:
Done [110b73a](https://github.com/red-hat-data-services/ods-ci/pull/1209/commits/110b73a6aa09d7ed1e07c3691624829a284a5a3b)